src/app/routes/home/getInitialData/index.js

```diff
@@ -14,8 +17,26 @@ const transformJson = pipe(
   filterGroupsWithoutStraplines,
 );
 
-export default async (path) => {
-  const { json, ...rest } = await fetchPageData(path);
+export default async ({ path, service, variant }) => {
+  const config = await getConfig(service, variant);
+
+  const hasRadioSchedule = pathOr(
+    false,
+    ['radioSchedule', 'hasRadioSchedule'],
+    config,
+  );
+
+  const radioScheduleOnFrontPage = pathOr(
+    false,
+    ['radioSchedule', 'onFrontPage'],
+    config,
+  );
+
```

Comment:
One thing we are not doing here is respecting the value of the toggle for radio schedules for this particular environment. We could import the static toggles json file and check that, but we are moving towards an approach where toggles come from an external endpoint, and are configured within the react application. So as is, if radio schedules is toggled off for an environment, we would still fetch the data for them on the server (and not render the component due to the toggles checks within the react application).